src/selectionHandler.ts

```diff
@@ -92,7 +97,17 @@ function resolveDetailOf(completion: any): string {
 
 async function handleImports(editor: TextEditor, completion: any) {
     let selection = editor.selection;
-    let completionSelection = new Selection(selection.active.translate(0, -completion.length), selection.active);
+    let lineDelta = 0;
+    let characterDelta = -completion.length;
+
+    if (completion.indexOf(EOL) > 0) {
```

Comment:
i've ment - pass the `position` that you have in `selectionHandler ` to `handleImports` and then :
instead of `let completionSelection = new Selection(selection.active.translate(lineDelta, characterDelta), selection.active)`
use `new Selection(position, editor.selection.anchor)`